### PR TITLE
immersed-vr: rename to immersed

### PR DESCRIPTION
**Pull request by Pandapip1**
## Description of changes

See https://github.com/repology/repology-rules/pull/851 for rationale

## Things done

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by haruki7049** [comment]
Are you saying that only the immersed-vr package in nixpkgs is not named immersed compared to other package managers?

**Comment by Pandapip1** [comment]
> Are you saying that only the immersed-vr package in nixpkgs is not named immersed compared to other package managers?

As far as I know, there's only one other package manager that has packaged it, but its name is arguably more correct, as upstream doesn't use the `-vr` suffix and there is no conflict with other packages.

**Comment by haruki7049** [comment]
Understood. Sure, we should match the naming of other package managers.

**Review by haruki7049 — APPROVED** [review]
LGTM!!

File: nixos/modules/module-list.nix
Comment (Aleksanaa) [inline nixos/modules/module-list.nix:1]: There's also a module alias in `nixos/modules/rename.nix`

Comment (Pandapip1) [reply nixos/modules/module-list.nix:1]: > This file defines some renaming/removing options for backwards compatibility
>
> It should ONLY be used when the relevant module can't define these imports
> itself, such as when the module was removed completely.
> See https://github.com/NixOS/nixpkgs/pull/61570 for explanation

A rename has been added in the renamed module: https://github.com/NixOS/nixpkgs/pull/341266/files#diff-ca2d952d1105185448a29a0e039dd2b09b8a33a22c4b3e4a733afa56967b588aR12-R23

**Comment by Pandapip1** [comment]
Rebased to fix merge conflict.

**Review by fpletz — APPROVED** [review]

**Pull request merged**